### DISCUSSION TITLE
【收集】第一批 Podcast 列表，赶快留下你最爱的栏目吧~

### DISCUSSION THREAD

**Discussion by booooodv** [booooodv/PodLetter#7] [Announcements]
如果可以，请留下你的理由~

希望置顶推荐，请单独发帖：【推荐】xxxx 节目

根据投票会尽快置顶。

**Comment by eddiehe99** [comment]
《[生活漫游指南](https://www.xiaoyuzhoufm.com/podcast/5e509e58418a84a046b2d5aa)》
理由：漫谈类科普播客，由于主持人的插科打诨，节目的欢乐程度很高。
